fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/api/content/registry/v1/BlockEntityRegistry.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.content.registry.v1;
+
+import net.minecraft.block.entity.BlockEntity;
+
+import net.fabricmc.fabric.impl.content.registries.ContentRegistryImpl;
+
+public final class BlockEntityRegistry {
+	private BlockEntityRegistry() {
+	}
+
+	public static void registerBlockEntity(Class<? extends BlockEntity> clazz, String name) {
```

Comment:
pass an identifier here

Reply:
block entities need a string. that identifies their name

Reply:
what if 2 mods register them with the same name? identifiers need to be used here

Reply:
it fails soft and skips registering the duplicate one. 
also, if you using identifiers, the block entity name would become `<namespace>:<name>` instead `<Name>`

Reply:
exactly, thus fixing the bug with duplicates.

Reply:
you are aware later api versions use identifiers in that way right?

Reply:
1.8.9 uses strings to identify entities, not identifiers. Versions 1.12 and above use identifiers iirc

Reply:
alright will set it to identifiers

Reply:
done